ApollosProject/apollos-apps
# Adds CampaignItemListFeature to ui-connected package

## DESCRIPTION
Companion PR to https://github.com/ApollosProject/apollos-templates/pull/31. Merge this PR first.
<img src="https://user-images.githubusercontent.com/2053547/79017834-4b51d900-7b40-11ea-856c-c6b8ee06d109.gif" width="30%" />

### What does this PR do, or why is it needed?
This just packages the `CampaignListItemFeature` that is usually used at the top of the home tab.

### What design trade-offs/decisions were made?

### How do I test this PR?
open the app. check the stories. run the tests.

Actually, tests on this PR wont run because of some mocking error in ui-connected that needs to be resolved. This PR is unmergable until it's resolved.
### What automated tests did you write?
👍 
## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
